bin/reload-upstream

```diff
@@ -58,10 +64,10 @@ function fetchdocs () {
 # finally, apply our references
 #
 
-fetchdocs docs/boshrelease https://github.com/logsearch/logsearch-boshrelease.git docs
-fetchdocs docs/shipper-boshrelease https://github.com/logsearch/logsearch-shipper-boshrelease.git docs
-fetchdocs docs/filters-common https://github.com/logsearch/logsearch-filters-common.git docs
-fetchdocs docs/workspace https://github.com/logsearch/workspace.git docs
+fetchdocs docs/boshrelease https://github.com/logsearch/logsearch-boshrelease.git docs getting-started-with-logsearch-workspace
```

Comment:
I disagree with this change from the perspective of "content and presentation are separate".

These sorts of content changes in the other repositories should be reviewed as a regular [compare/PR](https://github.com/logsearch/logsearch-boshrelease/compare/getting-started-with-logsearch-workspace), or, if inline diffs aren't particularly appropriate for the changes, by simply browsing the [new files](https://github.com/logsearch/logsearch-boshrelease/blob/getting-started-with-logsearch-workspace/docs/getting-started/creating-kibana-dashboards.md).

Since this repo is simply referencing the sub-project files, we shouldn't need to preview a whole new PR website every time one of the sub-projects is getting some docs love. If this were magically rewriting our content from those sub-projects in wildly unpredictable ways, it would be important for us to preview every change via changes like this line.

So, in my mind, our workflow should be:

0. Update docs in sub-project.
0. Sub-project PR for the docs change (if necessary) - use GitHub's built-in diff/previews for reviewing content changes.
0. Once merged, build server notifies this website build as a downstream project that it is affected.
0. This project gets rebuilt and artifact'd (and immediately, or another downstream build if we want to require manual review/publish for some reason, publishes it to our public site).